pkg/kn/commands/source/apiserver/apiserver_create.go

```diff
@@ -0,0 +1,72 @@
+// Copyright © 2019 The Knative Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package apiserver
+
+import (
+	"errors"
+
+	"github.com/spf13/cobra"
+
+	"knative.dev/client/pkg/kn/commands"
+	"knative.dev/client/pkg/kn/commands/source/sink"
+)
+
+func NewApiServerCreateCommand(p *commands.KnParams) *cobra.Command {
+	var apiServerUpdateFlags ApiServerSourceUpdateFlags
+	var sinkFlags sink.SinkFlags
+
+	cmd := &cobra.Command{
+		Use:   "create NAME --resources RESOURCE --accountname ACCOUNTNAME --sink SINK --mode MODE",
```

Comment:
`--accountName` ?

Reply:
whyt not just `--account` ?

Reply:
`--resource` instead of `--resources` ?

Reply:
I plan to use `--resources` to describe a list of resources because ApiServerSource supports to define the monitoring of multiple resources.